app/views/layouts/application.haml

```diff
@@ -24,7 +24,8 @@
     - if(@view_type.eql?("map"))
       = javascript_include_tag "markerclusterer.js"
     - props = topbar_props
-    = react_component("TopbarApp", props: props, prerender: true)
+    - cache props do
```

Comment:
That's correct. We thought this is a good starting point, so that we can deploy this and implement the async personal data fetching later. Initially we thought that it can be accomplished without an additional http request by just rendering the user specific things on client side, but as the data for client side rendering is also output by the `react_component` call, it gets cached as well and doesn't work any better.